src/Telegram.php

```diff
@@ -23,7 +23,6 @@
  */
 class Telegram
 {
-
```

Comment:
Some classes have an empty line here, others don't. To me no empty line looks better.
If there should be an empty line, should there also be one at the very end of the class?